ctapipe/io/hdf5tableio.py

```diff
@@ -72,7 +74,15 @@ class HDF5TableWriter(TableWriter):
 
     """
 
-    def __init__(self, filename, group_name, mode='w', root_uep='/', **kwargs):
+    def __init__(
+        self,
+        filename,
+        group_name,
+        add_prefix=False,
+        mode='w',
+        root_uep='/',
```

Comment:
what is a "uep" ?

Reply:
"The user entry point to the object tree attached to the HDF5 file is represented in the root_uep attribute". From http://www.pytables.org/usersguide/libref/file_class.html

Reply:
We want to get rid of `root_uep`. We want to merge it with the `group_name`.